README.md

```diff
@@ -619,6 +619,79 @@ the field's description which includes the spark type of the field.
 In order to write those types to BigQuery, use the ORC or Avro intermediate format, and have them as column of the
 Row (i.e. not a field in a struct).
 
+#### BigNumeric support
+BigQuery's BigNumeric has a precision of 76.76 (the 77th digit is partial) and scale of 38. Since this much precision
+and scale is beyond the supported 38 scale and 38 precision of spark's DecimalType, the BigNumeric DataType is
+converted into spark's [UserDefinedType](https://spark.apache.org/docs/1.4.0/api/java/org/apache/spark/sql/types/UserDefinedType.html).
+The BigNumeric data can be accessed via BigNumericUDT DataType which internally uses [java.math.BigDecimal](https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html) 
+to hold the BigNumeric data. The data can be read in either AVRO or ARROW formats.
+
+In order to write BigNumericUDT to BigQuery, use either ORC or PARQUET intermediate formats. Notice that the data gets
```

Comment:
Should we call out if there are any unsupported formats

Reply:
True, but we need to do it in another PR, not this one.

Reply:
I am explicitly mentioning (as a precautionary measure) that we do not support AVRO.